thirdeye/thirdeye-frontend/app/pods/components/detection-yaml/template.hbs

```diff
@@ -3,9 +3,10 @@
 </div>
 <div class="col-xs-12 {{if isEditMode "bottom-margin"}}">
   {{#unless isEditMode}}
-    <label for="select-metric" class="control-label te-label te-label--taller required">Can't find your metric?
+    <label for="select-metric" class="control-label te-label te-label--taller required">Can't find your metric? Import a Metric From
```

Comment:
Why splitting the sentence into two parts?
"Import a Metric From" is not a complete sentence.

Reply:
Here are some screenshots to show why the sentence looks split: 
<img width="1139" alt="Screen Shot 2019-07-31 at 10 25 03 AM" src="https://user-images.githubusercontent.com/10106733/62233529-9e527a00-b37d-11e9-853b-910d4e930e5a.png">
<img width="457" alt="Screen Shot 2019-07-31 at 10 25 11 AM" src="https://user-images.githubusercontent.com/10106733/62233552-a90d0f00-b37d-11e9-9971-dadd665f3c7d.png">

Reply:
"Ingraphs" links to the route for importing Ingraph, and "MySQL/Presto" for those metrics